test/unittests/test_dispatcher.py

```
@@ -0,0 +1,259 @@
+
+class TestDispatchFromMatch(unittest.TestCase):
+    """The dispatch + trio must fire from the orchestrator's match path."""
+
+    def _make_service(self):
+        bus = FakeBus()
+        svc = IntentService.__new__(IntentService)
+        svc.bus = bus
+        svc.config = {}
+        svc.pipeline_plugins = {}
+        svc._deactivations = defaultdict(list)
+        ut = MagicMock(); ut.transform.side_effect = lambda u, c: (u, c)
```

Ruff (0.15.18)

[error] Multiple statements on one line (semicolon)

(E702)